packages/slate-editor/src/modules/editor-v4/lib/useCursorInView.ts

```diff
@@ -34,7 +34,10 @@ function ensureCursorInView(
 
     const [currentNode] = EditorCommands.getCurrentNodeEntry(editor) || [];
 
-    if (Editor.isBlock(editor, currentNode) && Editor.isVoid(editor, currentNode)) {
+    if (
+        Editor.isBlock(editor, currentNode) ||
+        (Editor.isBlock(editor, currentNode) && Editor.isVoid(editor, currentNode))
```

Comment:
I don't understand this change. 

`Editor.isBlock(editor, currentNode) || Editor.isBlock(editor, currentNode) && Editor.isVoid(editor, currentNode)` is equivalent to just `Editor.isBlock(editor, currentNode)`.